### ISSUE TITLE
[A11y] Search Results - Grid - search field label

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1937]
### Operating System

Windows

### Browser

Edge

### Affected page

Search

### Admin user?

Yes

### Issue Description

The "Search within the results" field has two labels which are both visually hidden.  This causes a really weird label to be announced prior to the edit:  "Search by keyword Search within the results  edit..."

### Suggested Fix

One of the hidden labels should be removed - I'd recommend the "Search by keyword" label, because I don't think that's helpful considering what sighted users see.

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

This is easiest tested with a tool that tells you the accessible description/name/label.  In firefox you can do this pretty easily by right-clicking the search field and choosing "Inspect Accessibility Properties".  A screen reader will also make it obvious if there are two labels, but users unfamiliar with screen reader behavior may get confused - the placeholder text is also announced, at least by NVDA, after the rest of the information.  So it may sound like there are still duped labels when the accessible name is in fact fine.
